src/mlpack/methods/ann/layer/transposed_convolution.hpp

```diff
@@ -397,8 +342,11 @@ class TransposedConvolution
   //! Locally-stored transformed gradient parameter.
   arma::cube gradientTemp;
 
-  //! Locally-stored padding layer.
-  Padding<>* padding;
+  //! Locally-stored padding layer for Forward padding.
```

Comment:
It would be great if you could edit this to `for forward propagation` and the bolow one to `for back propagation`. :+1: